src/components/tipRecords/TipRecord.vue

```diff
@@ -110,23 +110,49 @@
             :src="tipPreviewImage"
             :onerror="`this.className+=' fail'`"
             :loading="`lazy`"
-            class="preview__image"
+            class="author_img preview__image"
           >
         </div>
       </div>
-      <div
-        v-else
-        class="tip__url"
-      >
-        <a
-          :href="tip.url"
-          :title="tip.url"
-          class="text-ellipsis"
-          target="_blank"
+      <div v-else>
+        <div v-if="tip.url">
+          <a
+            :href="tip.url"
+            :title="tip.url"
+            class="text-ellipsis"
+            target="_blank"
+            @click.stop
+          >
+            {{ tip.url }}
+          </a>
+        </div>
+        <div
+          v-if="tip.receiver"
+          class="tip__author"
+          :title="tip.receiver"
           @click.stop
         >
-          {{ tip.url }}
-        </a>
+          <TipInput
+            is-retip
+            :tip="tip"
+          />
+          <router-link :to="'/user-profile/' + tip.receiver">
+            <Avatar :address="tip.receiver" />
+            <div class="tip__author_name">
+              <span
+                v-if="tip.chainName"
+                class="chain__name"
+              >
+                {{ tip.chainName }}
+              </span>
+              <span
+                v-else
+                class="chain__name"
+              />
```

Comment:
```suggestion
              <span class="chain__name">
                {{ tip.chainName }}
              </span>
```
Does the empty text node break something in this case?